keys.go

```diff
@@ -203,6 +207,26 @@ func (c *Client) createKey(ctx context.Context, key Key) (*Key, error) {
 	return &keysResponse.Keys[0], nil
 }
 
+// TransferKeyToNewKeyRing method transfers a key associated with one key ring to another key ring
+// For more information please refer to the link below:
+// https://cloud.ibm.com/docs/key-protect?topic=key-protect-grouping-keys#transfer-key-key-ring
+func (c *Client) TransferKeyToNewKeyRing(ctx context.Context, keyID, newKeyRingID string) (*Key, error) {
```

Comment:
This is more of a `SetKeyRing` honestly. It will always put the new value so its a set.